src/pages/login.tsx

```diff
@@ -1,7 +1,124 @@
+import React, { useState, FormEvent } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import CustomButton from "../components/custombutton-large";
+
 export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleUsernameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    // Reset the form
+    setUsername("");
+    setPassword("");
+  };
+
   return (
-    <main>
-      <h1>Login Page</h1>
-    </main>
+    <div className="flex h-screen items-center justify-center">
+      <div className="rounded-md bg-white p-8 shadow">
+        <div className="flex items-center justify-center">
+          <Image
+            src="/images/repair_lab_logo.jpg"
+            alt="logo"
+            width={80}
+            height={80}
+          />
+        </div>
+        <div className="flex items-center justify-center">
+          <h1 className="m-4 text-2xl font-bold">Sign in</h1>
+        </div>
+        <CustomButton
+          onClick={() => {
+            // Handle Google login here
+            console.log("Continue with Google...");
+          }}
+          text="Continue with Google"
+        />
+
+        {/* Separate the "Login with Google" button with a custom color dashed line and "or" text */}
+        <div className="mb-4 flex items-center">
+          <div
+            className="border-t-2 border-dashed"
+            style={{ borderColor: "#098D85", flex: 1 }}

```

Comment:
Same comments for this file as with `register.ts`. Remove all of the `style` objects and use the equivalent tailwind classes please. LMK if you need help finding them.